api/AudioTrack.json

```diff
@@ -697,7 +697,9 @@
               "version_added": "6.1"
             },
             "safari_ios": {
-              "version_added": "7"
```

Comment:
Are you sure about this? `AudioTrack` is separate from MSE and I believe shipped much earlier, I suspect the existing data is correct.

Reply:
This particular property is a part of the spec for MSE.  I've confirmed that there's no support before Safari iOS 13 in BrowserStack.

Reply:
Oh, this is the `sourceBuffer` property! I didn’t expand the diff context 🙄